src/test/daml/Daml/Finance/Settlement/Test/BatchWithIntermediaries.daml

```diff
@@ -63,57 +62,51 @@ data TestParties = TestParties
 -- |          \   /              | securities             |
 -- | Central Security Depository |                        |
 -- +-----------------------------+------------------------+
-run : Bool -> Bool -> Script ()
-run bankAllocatesWithOwnHolding bankUsesCompatibleHoldingStandard = script do
+run : Bool -> Script ()
```

Comment:
can we improve the description to specify what it means for a bank to allocate with their own holding?

Reply:
I removed that case. Although it works, it is not something we would need to promote. It is much more efficient for the Bank to simply allocate its instruction towards the seller with a CreditReceiver.